### DISCUSSION TITLE
Kata 3 me sale incompleta.

### DISCUSSION THREAD

**Discussion by OscarIslasH** [LaunchX-InnovaccionVirtual/CursoIntroPython#184] [Q&A]
Tengo un problema al momento de abrir el folder de "Katas propias" copie y pegue las katas apenas las descargue, pero tengo el problema que al momento de ejecutar la kata 3, me sale incompleta como en imagen:

![Captura de pantalla 2022-02-12 051657](https://user-images.githubusercontent.com/99118114/153709249-bf9567b1-8213-410e-9935-c1c3b858890c.png)

Desdcargue nuevamente el archivo para reemplazarlo ya que crei que sin querer habia borrado codigo de la kata 3, sin embargo, seguia estando igual al reemplazarlo, lo borre y volvi a copiarlo, pero me sigue saliendo igial. Solo me sale completo si lo abro desde la carpeta de "CURSO DE PYTHON MAIN" el cual descargue, como se ve en la imagen:

![Captura de pantalla 2022-02-12 051746](https://user-images.githubusercontent.com/99118114/153709248-3db66bd7-39c4-4455-ad97-2933abe78a5a.png)

Si alguien pudiera ayudarme con que me salga completa la kata, ya que la resolvi desde la carpeta de "CURSO DE PYTHON MAIN" pero igual al pasarla a la de "Katas propias" sale esa parte incompleta :(
 
PD: Ya cerre los folders una y otra vez para que los cambios de la kata que sale mal, no se sobre escriban sobre los de la kata buena e inte hacer lo inverso, guardar los cambios de la buena sobre los de la mala pero no funciono

**Comment by Paulina-Ceciliano** [comment]
¿De qué manera copiaste los archivos? Directamente desde la carpeta, ¿o cómo?
Yo también cambié los archivos de carpeta. 
Entré directamente a la carpeta en el escritorio donde estaban guardados, seleccioné los archivos, los copié y los pegué en otra porque tenía un desastre.
![image](https://user-images.githubusercontent.com/88915301/153710034-2aa4c8eb-aac1-4ff8-b02f-fd50b05d2d62.png)

**Reply by AlbertoVillanuevaE** [reply]
Algo muy similar me pasó, prueba añadiendo un nuevo cuadro de markdown o de código en el espacio que falta, a mi luego de hacer esto me aparecía el texto faltante

**Comment by Stmark1** [comment]
Hola, no estan incompletas, estan redactadas así por que la intención es que tu escribas el codigo que falta.

**Reply by OscarIslasH** [reply]
No, el único problema que me sale es el tercero, los primeros dos se borraron, no me sale lo que debo hacer en los primeros dos problemas, si quieres revisa la kata, vas a ver qué faltan dos problemas que a mí no me salen, solo sale el tercer problema :(

**Reply by Stmark1** [reply]
Ah, ya entendí, en ese caso lo que puedes hacer es copiar manualmente el contenido de cada celda faltante a tu archivo, y luego guardarlo, de esa forma te aseguras de tener lo que te falta.

Para ver el contenido de una celda dale doble click, luego selecciona todo y le das copiar, creas la celda correspondiente en tu archivo y luego lo pegas. Si es markdown le das en la palomita que aparece en la parte superior derecha y se visualiza.

**Comment by alfredo-inzunza** [comment]
Así com menciona @Stmark1 , el archivo que tiene como terminación "Katas", es como un archivo entregable, alli vienen las instrucciones de lo que tienes que hacer (antes del bloque de código). Incluso como puedes ver, las líneas tienen un # (numeral) que las antecede, esto es porque son solo comentarios, esto lo ignora el programa, alli en los comentarios te indica que es lo que hay que hacer en cada bloque. 

Si de plano estás muy perdido, al final te muestra las soluciones, aunque lo ideal es que lo intentes resolver tu solo para practicar. 
Éxito.

**Reply by OscarIslasH** [reply]
Es que solo sale el tercer problema, no sale ni el primero ni el segundo, literalmente ví como se borraron cuando los estaba haciendo y no sé por qué, estaba mirando la pantalla y solamente se borró el código.

Si quieres revisa la kata 3 y compara la primera captura, solo me sale el tercer problema.

**Reply by alfredo-inzunza** [reply]
Ha, ya entendí a que te refieres. 
Yo lo que hago es abrir el "original" del repositorio de Fer que descargué y descromprimí en mi PC,  una vez abierto en VsCode le doy Guardar como... y ya lo guardo en la ruta en donde tengo mis Katas propias. Espero te sirva. Saludos
